### DISCUSSION TITLE
Flickr fun / funny Flickr

### DISCUSSION THREAD

**Discussion by srchild** [mcguirepr89/BirdNET-Pi#475] [General]
I received an Apprise notification via Telegram of an unusual species, Brant. I was at the back door, where the microphone is located, so thought to go outside and have a look.

Before doing so I brought up BirdNET-Pi on my phone, to remind me what Brant look like. Hmm, a bird with no wings and no beak. Then I went outside to look for her, but didn't find her. See photos attached.

![Screenshot_20220726-170830](https://user-images.githubusercontent.com/104400455/181992671-d177b163-7b83-47c0-86d8-eba5141ad23b.png)
![Screenshot_20220726-170819](https://user-images.githubusercontent.com/104400455/181992870-c1f9e468-b75d-4b3e-808b-3431c5f434f2.png)

The serious point here is that there appears to be a risk that BirdNET-Pi might retrieve inappropriate images, which could be a problem in an installation in some public settings, for example if children may view the results. Perhaps there needs to be some way to set "safe-search" or similar when retrieving images using Flickr API?

**Comment by ehpersonal38** [comment]
LOL! Sorry you couldn't find her.

I looked through the Flickr docs and it seems that safe search is by default on, even for the API. I am also able to find that same image on the Flickr website search with "safe search" on.

As for preventing this in the future... I'm not sure. I'd hate to have to hard code these edge cases in, e.g. turning the query "Brant" into "Brant bird," but it may have to be done.

**Comment by Phaeton** [comment]
I had a fun one the other day as well. While not an adult content concern, was definitely not what I wanted to find in my backyard. 

<img width="323" alt="Sora" src="https://user-images.githubusercontent.com/808865/182028639-02de8460-a69f-42a0-a6ba-5bf0871232fb.png">

I get a lot of train pictures when rails are detected and chimney for a chimney swift, but I think my favorite has been a Merlin. Was quite funny to see a wizard image be returned.

**Reply by matthew73210** [reply]
Had a chuckle myself when this appeared for a black grouse that sometimes calls during the night 
![download](https://user-images.githubusercontent.com/19491784/182048903-2d95bd78-79b6-4bed-98b6-9ef804ecaf56.jpeg)

**Reply by ehpersonal38** [reply]
😆 that one was a paid sponsorship

**Comment by ehpersonal38** [comment]
Okay, let's try this commit out: 3c75f63d51218234caa7c105ffc5628e4fe580db

report back if anything changes for the worse and I'll try something else

**Comment by matthew73210** [comment]
Flickr strikes again 🥇 
<img width="894" alt="Screenshot 2022-08-15 at 12 36 56 PM" src="https://user-images.githubusercontent.com/19491784/184621022-40837f8b-3878-4c3b-b5b4-6d44586a8d9c.png">
The common house martin has evolved

**Comment by Datore7** [comment]
Have the same owl here for Common House-Martin,

However, it only seems to show the wrong bird image where small image shows ie. at "Today's Discoveries" and "Recent Discovery",
but when looking at "Best Recordings" and select the bird with the wrong image, all the larger images are correct.
